src/omv/ports/mimxrt/sensor.c

```diff
@@ -24,14 +24,21 @@
 #include "framebuffer.h"
 #include "unaligned_memcpy.h"
 
+#define DMA_LENGTH_ALIGNMENT     (8)
+#define SENSOR_TIMEOUT_MS        (3000)
+
 // Sensor struct.
 sensor_t sensor = {};
 extern uint8_t _line_buf[OMV_LINE_BUF_SIZE];
 
+#if defined(OMV_CSI_DMA)
+static edma_handle_t CSI_EDMA_Handle[OMV_CSI_DMA_CHANNEL_COUNT];
+static int src_inc, src_size, dest_inc_size;
+#endif
```

Comment:
There's a way to add variables like those directly in `sensor_t` struct, see `OMV_SPI_PORT_BITS`